sigstore/_cli.py

```diff
@@ -669,11 +739,16 @@ def _sign(args: argparse.Namespace) -> None:
                 print(result.cert_pem, file=io)
             print(f"Certificate written to {outputs['cert']}")
 
+        if outputs["rekor_bundle"] is not None:
```

Comment:
I changed these back from:
```python
if "rekor_bundle" in outputs:
```
This is because when we don't write to an output, the entry still gets added to the map, the file is just set to `None`.

Reply:
Makes sense! I've re-fixed https://github.com/sigstore/sigstore-python/pull/465#discussion_r1086729138 based on that.